docs/getting_access/non_standard.md

```diff
@@ -0,0 +1,235 @@
+
+```bash
+"error: execve(): orted: No such file or directory".
+```
+
+This wrapper script must be named `orted`, and must be located in a path that is listed in `$PATH`.
+
+For example, it can be placed in `~/bin/orted`, and you can add `export PATH=$HOME/bin:$PATH` to your `~/.bashrc` login script.
+
+Contents of ~/bin/orted:
+
+```bash
+#!/bin/bash
+
+# first remove path to this orted wrapper from $PATH, to avoid infinite loop
+orted_wrapper_dir=$(dirname $0)
+export PATH=$(echo $PATH | tr ':' '\n' | grep -v $orted_wrapper_dir | tr '\n' ':')
+
+~/bin/cvmfsexec_eessi.sh orted "$@"
+```
+
+Again, do make sure that also this `orted` wrapper script is executable:
+
+```bash
+chmod u+x ~/bin/orted
+```
+
+If not, you will likely run into an error that starts with:
+
+
+```bash
+An ORTE daemon has unexpectedly failed after launch ...
+```
+
+### Slurm job script
+
+The `cvmfsexec_eessi.sh` can be used inside a Slurm job script inside your HPC system to initialize the EESSI environment in a subshell which the EESSI CernVM-FS repository is mounted. Thus, you will be able to load any module from EESSI you might need.
+
+Example job script:
+
+```bash
+#!/bin/bash
+#SBATCH --ntasks=4
+#SBATCH --ntasks-per-node=2
+#SBATCH --cpus-per-task=1
+#SBATCH --time=5:0:0
+#SBATCH --export=None
+#SBATCH --mem=30000M
+~/bin/cvmfsexec_eessi.sh << EOF
+source /cvmfs/software.eessi.io/versions/2023.06/init/bash
+module load TensorFlow/2.13.0-foss-2023a
+export SLURM_EXPORT_ENV=HOME,PATH,LD_LIBRARY_PATH,PYTHONPATH
+mpirun -np 4 python test.py
+EOF
+
+```
+You can see the original blog post on how they used this solution in Deucalion [here](https://www.eessi.io/docs/blog/2024/06/28/espresso-portable-test-run-eurohpc/#running-espresso-on-deucalion-via-eessi-cvmfsexec). 
+
+## Via `squashfs` +  cvmfs's `shrinkwrap` utility
+
+CernVM-FS provides the Shrinkwrap utility, allowing users to create a portable snapshot of a CVMFS repository. This can be exported and distributed without the need of a CVMFS client or network access.
+
+To create an export of EESSI in user space, you will first need to create the config file `software.eessi.io.config`:
+
+```bash
+CVMFS_REPOSITORIES=software.eessi.io
+CVMFS_REPOSITORY_NAME=software.eessi.io
+CVMFS_CONFIG_REPOSITORY=cvmfs-config.cern.ch
+CVMFS_SERVER_URL='http://aws-eu-west-s1-sync.eessi.science/cvmfs/software.eessi.io'
+CVMFS_HTTP_PROXY=DIRECT # Avoid filling up any local squid's cache
+CVMFS_CACHE_BASE=/tmp/shrinkwrap
+CVMFS_KEYS_DIR=/etc/cvmfs/keys/eessi.io # Need to be provided for shrinkwrap
+CVMFS_SHARED_CACHE=no # Important as libcvmfs does not support shared caches
+CVMFS_USER=cvmfs
+CVMFS_UID_MAP=uid.map
+CVMFS_GID_MAP=gid.map
+
+```
+You will need to create the files `uid.map` and `gid.map` with the respective value you will use preceded by a `*`. For example, assuming UID 1000, set the two following files:
```

Comment:
I would explain to people that they _probably_ want to use the current user id and primary group, so
```bash
$ echo '*' $(id -u $USER) > uid.map
```
and
```bash
$ echo '*' $(id -g $USER) > uid.map
```